ios/FluentUI/Notification/NotificationTokens.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Pre-defined styles of the notification
+@objc public enum MSFNotificationStyle: Int, CaseIterable {
+    case primaryToast
```

Comment:
these should be documented too right?

Reply:
Not seeing any documentation here still.